packages/electron/src/common/ipc/IpcRpc.ts

```diff
@@ -0,0 +1,25 @@
+import { Event } from 'electron';
+
+export interface IpcRpcCaller {
+    invoke<T>(event: string, ...args: unknown[]): Promise<T>;
+}
+
+export interface IpcRpcHandler {
+    on(event: string, callback: (event: Event, ...args: any[]) => Promise<any>): this;
```

Comment:
can we please ignore any if we want to keep it? Please take a look into Github warnings

Reply:
changed possible `any` to `unknown` and added skips for the left